### DISCUSSION TITLE
ListBox等控件使用VirtualPanel排列大量数据问题

### DISCUSSION THREAD

**Discussion by ycode-lin** [BYJRK/DotNet-Discussions#172] [WPF 开发]
你好，请问有遇到过使用ListBox\ListView 使用 VirtualPanel 渲染大量数据的场景吗? 我遇到一个需求就是: 既要WrapPanel的可变渲染方向，又要使用类似VirtualStackPanel的虚拟化技术减轻加载压力，使渲染不会卡UI。请问这种场景下一般是怎么解决的? 重新实现一个"VirtualWarpPanel"吗(难度有点大)? 还是有比较成熟Nuget包解决了这个需求(使用过VirtualizingWrapPanel，效果不是很理想)?
 期待你的答复! 感谢

**Comment by CodingOctocat** [comment]
VirtualizingWrapPanel 为什么不理想？你需要 UWP 的那种 GridView 的效果吗？
